Description: Use for SO-version also the minor version.
Author: Anton Gladky <gladk@debian.org>
Last-Update: 2017-05-20

--- gl2ps-1.4.0.orig/CMakeLists.txt
+++ gl2ps-1.4.0/CMakeLists.txt
@@ -124,7 +124,7 @@ if(OPENGL_FOUND)
   set_target_properties(shared PROPERTIES OUTPUT_NAME gl2ps)
   set_target_properties(shared PROPERTIES
     VERSION ${GL2PS_MAJOR_VERSION}.${GL2PS_MINOR_VERSION}.${GL2PS_PATCH_VERSION}
-    SOVERSION ${GL2PS_MAJOR_VERSION})
+    SOVERSION ${GL2PS_MAJOR_VERSION}.${GL2PS_MINOR_VERSION})
   if(MSVC)
     set_target_properties(shared PROPERTIES COMPILE_FLAGS "-DGL2PSDLL -DGL2PSDLL_EXPORTS")
   endif(MSVC)
